internal/pkg/template/workload.go

```diff
@@ -341,6 +341,12 @@ type ObservabilityOpts struct {
 	Tracing string // The name of the vendor used for tracing.
 }
 
+// DeploymentConfiguraitonOpts holds values for MinHealthyPercent and MaxHealthyPercent.
+type DeploymentConfigurationOpts struct {
+	MinHealthyPercent int // MinHealthyPercent parameter represents an upper limit on the number of tasks in a service
+	MaxHealthyPercent int // MaxHealthyPercent represents a lower limit on the number of tasks in a service
```

Comment:
I feel like `MaxPercent` is a better name for this field for its real purpose. Also,
```suggestion
         MinHealthyPercent int // MinHealthyPercent represents the lower limit on the number of tasks that should be running for a service during a deployment or when a container instance is draining.
	MaxPercent int // MaxPercent represents the upper limit on the number of tasks that should be running for a service during a deployment or when a container instance is draining.
```